### ISSUE TITLE
FFNet agent decisions are not changing with workload changes

### ISSUE THREAD

**Issue by asmaalrawi** [geopm/geopm#3845]
**Describe the bug**
I tried to use a neural net trained on geopmbench to test ffnet inference on region characterization while running geopmbench. I expected the output of ffnet agent to show at least 95% of trace samples identifying the correct region with >=95% probability. Instead, region probabilities were largely static.

**GEOPM version**
3.2.0

**Expected behavior**
Output logits of ffnet reflect changes in workload behavior

**Actual behavior**
Logits that are output in trace are largely static.

**Additional Context**
FFNet integration tests with skip decorators reproduce this issue. The skip decorators should be removed and tests should pass when this issue is fixed.
